### PR TITLE
Code Review Lab

### PR DESCRIPTION
**Pull request by abemille**

### CODE REVIEW

**Comment by abemille** [comment]
Get rid of print statement in countOccurrences().  Add unit tests.  Add comments.

File: Exercise1.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+     * @param args
+     * 
+     *            Write an example that counts the number of times a particular
+     *            character, such as e, appears in a file. The character can be
+     *            specified at the command line. You can use xanadu.txt as the
+     *            input file.
+     */
+    public static void main(String[] args) {
+
+        Path textFile = Paths.get("xanadu.txt");
```
Comment (venigallav) [inline Exercise1.java]: Its better to assign file name to a final String and use it as a parameter

**Comment by abemille** [comment]
Added final modifier to path/file name in Exercise1.java.  Note that print statement was removed from countOccurrences() prior to initial commit.  (Since this was a simple change, I took care of this during our code review.)  By adding final and removing the print statement from countOccurrences(), I have made the two changes required for this homework.

**Comment by gingeraltoids** [comment]
This has been reviewed.  Good job.

**Pull request closed without merging**